services/app-api/utils/constants/constants.ts

```diff
@@ -20,3 +20,15 @@ export const buckets = {
   FORM_TEMPLATE: "formTemplates",
   FIELD_DATA: "fieldData",
 };
+
+// REPORTS
+
+export const reportTables = {
+  MCPAR: process.env.MCPAR_REPORT_TABLE_NAME!,
+  MLR: process.env.MLR_REPORT_TABLE_NAME!,
+};
+
+export const reportFormBuckets = {
```

Comment:
Nit: Would probably name either `reportBuckets` to match `reportTables` or `formBuckets` to match the env variable. Not sure why the env variable is named that but 🤷.